packages/plugins/i18n/admin/src/index.ts

```diff
@@ -74,6 +78,14 @@ export default {
       return actions;
     });
 
+    contentManager.apis.addDocumentAction((actions: DocumentActionComponent[]) => {
+      // TODO add to the right place in the list
+      // actions.splice(indexOfPublishAction, 0, BulkPublishAction);
```

Comment:
forgotten about?

Reply:
Oops yes and still not sure how to put this in the right place. any ideas? I'll go back to it